nimbus-jose-jwt_aws-kms-extension/src/main/java/com/nimbusds/jose/aws/kms/crypto/impl/KmsAsymmetricSigningCryptoProvider.java

```diff
@@ -66,6 +66,16 @@ public abstract class KmsAsymmetricSigningCryptoProvider extends BaseJWSProvider
 
     public static final Map<JWSAlgorithm, String> JWS_ALGORITHM_TO_MESSAGE_DIGEST_ALGORITHM =
             ImmutableMap.<JWSAlgorithm, String>builder()
+                    .put(JWSAlgorithm.RS256, MessageDigestAlgorithms.SHA_256)
+                    .put(JWSAlgorithm.RS384, MessageDigestAlgorithms.SHA_384)
+                    .put(JWSAlgorithm.RS512, MessageDigestAlgorithms.SHA_512)
+                    .put(JWSAlgorithm.PS256, MessageDigestAlgorithms.SHA_256)
+                    .put(JWSAlgorithm.PS384, MessageDigestAlgorithms.SHA_384)
+                    .put(JWSAlgorithm.PS512, MessageDigestAlgorithms.SHA_512)
+                    .put(JWSAlgorithm.ES256, MessageDigestAlgorithms.SHA_256)
+                    .put(JWSAlgorithm.ES384, MessageDigestAlgorithms.SHA_384)
+                    .put(JWSAlgorithm.ES512, MessageDigestAlgorithms.SHA_512)
+                    // backwards compatibility for KMS-defined algorithm strings
```

Comment:
Nit: Let's update this comment to say something like: `// Compatibility for KMS-defined algorithm strings`, since we have decided to support these algorithm names for the long-term.

Reply:
👍